audb/core/load.py

```diff
@@ -844,24 +864,39 @@ def load(
 
             db_is_complete = _database_is_complete(db)
 
-            # filter tables
-            requested_tables = filter_tables(tables, deps.table_ids)
+            # filter tables (convert regexp pattern to list of tables)
+            requested_tables = filter_tables(tables, list(db))
+
+            # add/split into misc tables used in a scheme
+            # and all other (misc) tables
+            requested_misc_tables = _misc_tables_used_in_scheme(db)
+            requested_tables = [
+                table for table in requested_tables
+                if table not in requested_misc_tables
+            ]
 
             # load missing tables
             if not db_is_complete:
-                cached_versions = _load_tables(
-                    requested_tables,
-                    backend,
-                    db_root,
-                    db,
-                    version,
-                    cached_versions,
-                    deps,
-                    flavor,
-                    cache_root,
-                    num_workers,
-                    verbose,
-                )
+                for _tables in [
```

Comment:
Is it required that we load the misc tables used in a scheme first?

Reply:
Yes, because `_load_tables()` might try to load tables in parallel if `num_workers > 1` which fails if it tries to load a table that uses a scheme defined in a misc table.
The easiest workaround is to call it twice, first for misc tables used in a scheme and afterwards for all other (misc) tables.

Reply:
I think without https://github.com/audeering/audformat/pull/274 it's still not safe, as we may have two misc tables that are used in schemes and rely on each other.

Reply:
True, I thought this was already forbidden ;)

So maybe we wait here until we have a bug fix release of `audformat`?

Reply:
Agree

Reply:
This was forbidden in `audformat` v0.15.1 and should no longer be an issue.